scripts/clean_nested_workspaces.sh

```diff
@@ -7,7 +7,7 @@ set -eu -o pipefail
 
 echo_and_run() { echo "+ $@" ; "$@" ; }
 
-readonly workspaceRoots=("e2e" "examples" "packages")
+readonly workspaceRoots=("e2e" "packages")
```

Comment:
you can nuke this helper script as well. with only a few nested workspaces it is easy enough to clean manually